src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+/**
+ * Represents the main driver of <code>Duke</code>
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructs an instance of <code>Duke</code>.
+     * @param filePath File path to file storing tasks
+     */
+    public Duke(String filePath) {
+        this.storage = new Storage(filePath);
+        this.ui = new Ui();
+        try {
+            this.tasks = new TaskList(this.storage.loadSave());
+        } catch (Exception e) {
```

Comment:
Might be better to catch specific exceptions like IOException or FileNotFoundException?